bin/add_celltypes_to_sce.R

```diff
@@ -67,6 +70,33 @@ option_list <- list(
     type = "character",
     help = "Path to file containing the reference for assigning consensus cell type labels",
     default = ""
+  ),
+  make_option(
+    opt_str = c("--consensus_validation_ref"),
+    type = "character",
+    help = "Path to TSV file mapping consensus validation groups to consensus labels for counting normal reference cells intended as input to inferCNV",
+    default = ""
+  ),
+  make_option(
+    opt_str = c("--diagnosis_celltype_ref"),
+    type = "character",
+    help = "Path to TSV file mapping broad diagnoses to consensus validation groups for counting normal reference cells intended as input to inferCNV.
+    This file should have columns `diagnosis_group` and `celltype_groups` where the latter is a comma-separated list of consensus validation groups",
+    default = ""
+  ),
+  make_option(
+    opt_str = c("--diagnosis_groups_ref"),
+    type = "character",
+    help = "Path to TSV file mapping broad diagnoses to individual diagnoses for counting normal reference cells intended as input to inferCNV.
+    This file should have columns `diagnosis_group` for the broad diagnosis and `submitted_diagnosis` with individual diagnoses in ScPCA",
+    default = ""
+  ),
+  make_option(
+    opt_str = c("--reference_cells_file"),
+    type = "character",
```

Comment:
I prefer defaults to failure. But I can be overruled.

```suggestion
    type = "character",
    default = "reference_cell_count.txt"
```